geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/package.scala

```diff
@@ -16,6 +16,7 @@ package object hbase {
     val CoprocessorUrl           : SystemProperty = SystemProperty("geomesa.hbase.coprocessor.url")
     val CoprocessorMaxThreads    : SystemProperty = SystemProperty("geomesa.hbase.coprocessor.maximize.threads", "true")
     val WriteBatchSize           : SystemProperty = SystemProperty("geomesa.hbase.write.batch")
+    val WriteFlushTimeout        : SystemProperty = SystemProperty("geomesa.hbase.write.flushTimeout")
```

Comment:
could you add this to the docs [here](https://raw.githubusercontent.com/locationtech/geomesa/main/docs/user/hbase/configuration.rst) ?

Reply:
Yep